src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java

```diff
@@ -137,16 +139,41 @@ public void init() throws IOException {
     @Component
     private class MongoDbValues {
         @Getter
-        @Value("${spring.data.mongodb.host}")
-        private String host;
-
-        @Getter
-        @Value("${spring.data.mongodb.port}")
-        private String port;
+        @Value("${spring.data.mongodb.uri}")
+        private String uri;
 
         @Getter
         @Value("${spring.data.mongodb.database}")
         private String database;
+
+        @PostConstruct
+        public void init() throws IOException {
+            if (uri != null) {
+                String secret = extractSecretFromUri(uri);
+                String modifiedUri = uri.replaceFirst(secret, "hidden_secret");
```

Comment:
Wasnt it uri that can contain multiple "uris". This won't work if the user is providing multiple replicaset instances. Right?

Reply:
A uri may only contain one username and password.

Reply:
An uri contains one username and password but may contain multiple hosts. See examples in application properties.

Reply:
Alright :)